<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 239, 239);

QPushButton {
background-color: rgb(87, 137, 183);
selection-color: rgb(255, 170, 127);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
selection-background-color: rgb(66, 111, 153);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>770</height>
     </rect>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QPushButton" name="chick_nug_button">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Chicken
Nuggets</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hotdog_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
selection-background-color: rgb(66, 111, 153);</string>
      </property>
      <property name="text">
       <string>Hot Dog</string>
      </property>
     </widget>
     <widget class="QPushButton" name="admin_button">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QLabel" name="food_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Food</string>
      </property>
     </widget>
     <widget class="QPushButton" name="soda_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Soft Drink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkout_button">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>690</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Checkout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="salad_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Salad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chick_sand_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Chicken
Sandwich</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hamburger_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
selection-color: rgb(255, 170, 127);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
selection-background-color: rgb(66, 111, 153);</string>
      </property>
      <property name="text">
       <string>Hamburger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="milk_shake_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>520</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Milk Shake</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tea_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>520</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Tea</string>
      </property>
     </widget>
     <widget class="QLabel" name="drink_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Drinks</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fries_button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Fries</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="item_scroll">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>90</y>
        <width>391</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>389</width>
         <height>579</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>581</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="scroll_layout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="admin_page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>521</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>This is the admin page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="admin_button_2">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>50</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="toppings_page">
     <widget class="QLabel" name="food_label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>800</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>What toppings would you like on your hamburger?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="done_button">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>680</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 137, 183);
color: rgb(255, 255, 255);
alternate-background-color: rgb(87, 137, 183);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout2"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout3"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout4"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>140</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout5"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout6"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>330</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout10"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout8"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>330</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout9"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout7"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout11"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>520</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout15"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>520</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout13"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>520</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout14"/>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_15">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>520</y>
        <width>191</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout12"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
